console-subscriber/examples/app.rs

```diff
@@ -5,7 +5,25 @@ async fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync>> {
     console_subscriber::init();
 
     let task1 = tokio::spawn(spawn_tasks(1, 10));
-    let task2 = tokio::spawn(spawn_tasks(10, 100));
+    let task2 = tokio::spawn(spawn_tasks(10, 30));
+
+    // spawn optional extras from CLI args
```

Comment:
it might be kind of nice to have a comment at the top of the file (and/or a `--help` flag?) that mentions the CLI args, and summarizes the kind of bug that each one represents, so that people trying out the console with the example app know what they're doing?

not a blocker though!